tests/test_quantinuum_cancel.py

```diff
@@ -0,0 +1,28 @@
+import types
+
+import metriq_gym.quantinuum.job as qjob
+
+
+class DummyJobs:
+    def __init__(self):
+        self.cancel_called_with = None
+
+    def cancel(self, ref):
+        self.cancel_called_with = ref
+        return None
+
+    def results(self, _):
+        return []
+
+
+def test_cancel_calls_qnexus_jobs_cancel(monkeypatch):
```

Comment:
I am surprised this passes despite the cancel method is calling `_ref()`, which doesn't exist